docs-content/getting-started/3_client-sdk/7_replay-configuration/privacy.md

```diff
@@ -39,7 +39,18 @@ For sensitive input fields that your team would like to ignore user input for, y
 
 ## Default Privacy Mode
 
-By default, Highlight will obfuscate any text or input data that matches commonly used Regex expressions and input names of personally identifiable information. This offers a base level protection from recording info such as addresses, phone numbers, social security numbers, and more. It will not obfuscate any images or media content. It is possible that other, non PII text is obfuscated if it matches the expressions for larger number, or contact information on the site. If you want to turn this off, you can set `privacySetting` to `none` when calling [`H.init()`](../../../sdk/client.md#Hinit). 
+By default, Highlight will obfuscate any text or input data that matches commonly used Regex expressions and input names of personally identifiable information. This offers a base level protection from recording info such as addresses, phone numbers, social security numbers, and more. It will not obfuscate any images or media content. It is possible that other, non PII text is obfuscated if it matches the expressions for larger number, or contact information on the site. If you want to turn this off, you can set `privacySetting` to `none` when calling [`H.init()`](../../../sdk/client.md#Hinit).
+
```

Comment:
could we add a note that the above behavior is only for v8+ ? in case someone is wondering why they aren't obfuscating, if they are on an old client